modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/FDSDataRenderers/EntryStatusDataRenderer.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import React, {useEffect, useState} from 'react';
+
+import '../../../../object_entries/ViewObjectEntries.scss';
+
+import {fetch, sub} from 'frontend-js-web';
+
+function getDisplayType(status: string) {
+	switch (status) {
+		case 'approved':
+			return 'success';
+		case 'pending':
+		case 'scheduled':
+			return 'info';
+		case 'expired':
+			return 'warning';
+		default:
+			return 'secondary';
+	}
+}
+
+export default function EntryStatusDataRenderer({
```

Comment:
Could we use object prefix here? So `ObjectEntryStatusDataRenderer`. Remenber to rename the file name.

Reply:
done